_posts/2021-05-12-244.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Cockpit 244
+author: kkoukiou
+date: 2021-05-12
+tags: cockpit linux
+slug: cockpit-244
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 244 and Cockpit-Machines 244.
+
+## Shell: No opportunistic sudo invocation
+
+Cockpit will no longer try to run sudo on the very first login, and you have to
+explicitly turn on administrative access once for each browser profile that you
+use to access Cockpit. This prevents cockpit's use of sudo being noisy in the
+journal.
+
+![Sudo error messages](/images/sudo-error-messages.png)
```

Comment:
It seems to me that the screenshot does not match the description. The PR has this additional paragraph: "In addition, Cockpit now shows sudo's exact messages in the "Administrative access" dialog.", which is the explanation for the screenshot. Can you please add that?

Reply:
Ups right, sorry.

Reply:
Oh, I was assuming we wouldn't have a acreenshot. But if we would, I agree it should be something else. Although this one has strong https://xkcd.com/838/ vibes...

![incident](https://user-images.githubusercontent.com/10246/117935594-2e1fa900-b304-11eb-9b4e-10d8b2c9d58d.png)